seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1094,7 +1094,7 @@
         "<p class=\"red-text\">Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>"
       ],
       "tests": [
-        "assert($(\"img\").hasClass(\"smaller-image\"), 'message: Your <code>img</code> element should have the class <code>smaller-image</code>.');",
+        "assert(code.match(/<img\\s*?class\\s*?=\\s*?(\"|')?smaller-image\\s*?(\"|')?\\s*?src\\s*?=\\s*?(\"|')?https?:\\/\\/bit.ly\\/fcc-relaxing-cat\\s*?(\"|')?>/g) && (/<img\\s*?class\\s*?=\\s*?(\"|')?smaller-image\\s*?(\"|')?\\s*?src\\s*?=\\s*?(\"|')?https?:\\/\\/bit.ly\\/fcc-relaxing-cat\\s*?(\"|')?>/g.exec(code)[1] == /<img\\s*?class\\s*?=\\s*?(\"|')?smaller-image\\s*?(\"|')?\\s*?src\\s*?=\\s*?(\"|')?https?:\\/\\/bit.ly\\/fcc-relaxing-cat\\s*?(\"|')?>/g.exec(code)[2]) && (/<img\\s*?class\\s*?=\\s*?(\"|')?smaller-image\\s*?(\"|')?\\s*?src\\s*?=\\s*?(\"|')?https?:\\/\\/bit.ly\\/fcc-relaxing-cat\\s*?(\"|')?>/g.exec(code)[3] == /<img\\s*?class\\s*?=\\s*?(\"|')?smaller-image\\s*?(\"|')?\\s*?src\\s*?=\\s*?(\"|')?https?:\\/\\/bit.ly\\/fcc-relaxing-cat\\s*?(\"|')?>/g.exec(code)[4]), 'message: Your <code>img</code> element should have the class <code>smaller-image</code>.');",
```

Comment:
I don't understand why you have repeated the same regex 5 times?

For `"` you need to escape `\` and `"`.

If I do `<img class='           smaller-image' src="            https://bit.ly/fcc-relaxing-cat "           >` the regex will fail

If I do `<img src="https://bit.ly/fcc-relaxing-cat" class="smaller-image"` the regex will fail.